components/movementsensor/rtkutils/ntrip.go

```diff
@@ -31,6 +35,34 @@ type NtripConfig struct {
 	NtripUser            string `json:"ntrip_username,omitempty"`
 }
 
+// Sourcetable struct contains the stream.
+type Sourcetable struct {
+	Streams []Stream
+}
+
+// Stream contrains a stream record in sourcetable.
+// https://software.rtcm-ntrip.org/wiki/STR
+type Stream struct {
+	MP             string   // Datastream mountpoint
+	Identifier     string   // Source identifier (most time nearest city)
+	Format         string   // Data format (https://software.rtcm-ntrip.org/wiki/STR#DataFormats)
+	FormatDetails  string   // Specifics of data format (https://software.rtcm-ntrip.org/wiki/STR#DataFormats)
+	Carrier        int      // Phase information (https://software.rtcm-ntrip.org/wiki/STR#Carrier)
+	NavSystem      []string // Navigation System (https://software.rtcm-ntrip.org/wiki/STR#NavigationSystem)
+	Network        string   // Network name (https://software.rtcm-ntrip.org/wiki/NET)
+	Country        string   // ISO 3166 country code (https://en.wikipedia.org/wiki/ISO_3166-1)
+	Latitude       float32  // Position, Latitude in degree
+	Longitude      float32  // Position, Longitude in degree
+	Nmea           bool     // Caster requires NMEA input (1) or not (0)
+	Solution       int      // Generated by single base (0) or network (1)
+	Generator      string   // Generating soft- or hardware
+	Compression    string   // Compression algorithm
+	Authentication string   // Access protection for data streams None (N), Basic (B) or Digest (D)
+	Fee            bool     // User fee for data access: yes (Y) or no (N)
+	BitRate        int      // Datarate in bits per second
+	Misc           string   // Miscellaneous information
```

Comment:
Nice comments, but maybe a bit more wording on what each parameter is with the url.